c7n/resources/support.py

```diff
@@ -29,3 +29,7 @@ class resource_type(object):
         name = 'displayId'
         date = 'timeCreated'
         dimension = None
+
+    @classmethod
+    def get_arns(self, resources):
+        return ["arn:::{caseId}".format(**r) for r in resources]
```

Comment:
this should include `support` for the service and a valid value `account_id` id. Account arns don't because they are global scoped.